api-admin/src/main/java/bio/terra/pearl/api/admin/controller/GlobalExceptionHandler.java

```diff
@@ -76,13 +75,21 @@ protected static ResponseEntity<ErrorReport> buildErrorReport(
     for (Throwable cause = ex.getCause(); cause != null; cause = cause.getCause()) {
       causes.append("\nCause: ").append(cause);
     }
-    log.info(
-        "{}\nRequest: {} {} {}",
-        causes,
-        request.getMethod(),
-        request.getRequestURI(),
-        statusCode.value(),
-        ex);
+
+    String logString =
+        String.format(
+            "%s%nRequest: %s %s %s",
+            causes, request.getMethod(), request.getRequestURI(), statusCode.value());
+
+    switch (statusCode) {
```

Comment:
nit: I generally avoid `switch` except in extreme cases because I can never remember the fallthrough/break/default rules across languages.

this code might be cleaner if you either made some static Lists 
`public static final List ERROR_LOG_STATUSES = List.of(...)`
and then used `includes` to branch.  Alternatively, you could use the `statusCode.is5xxServerError()`for branching

Reply:
@devonbush 
Just curious, why are you negative on `switch`? Is that broadly frowned on these days, or a personal preference? I did not see any info on a quick search.

To my eye the alternatives seem harder to read, and this switch makes it clear how to add log customizations for other response statuses, but I could be missing something.

Reply:
There is also this style:
```    switch (statusCode) {
      case INTERNAL_SERVER_ERROR -> log.error(logString, ex);
      case BAD_REQUEST, UNAUTHORIZED, FORBIDDEN, NOT_FOUND -> log.info(logString, ex);
      default -> log.warn(logString, ex);
    }
```

Reply:
It was frowned on at the mid-sized company I worked for ~10 years ago, but otherwise just a personal preference.  I just have a mental block where no matter how many times I reread the `switch` syntax rules for Java/JS, I can't remember them offhand, and so a `switch` statement always slows me down and makes me worried.

Reply:
No opinion on which of the two switch syntaxes is best -- whatever you think is best

Reply:
@devonbush 
Can you re-review? While we were discussing the switch statements I did some manual testing which led to an update.